easybuild/easyconfigs/p/PSI/PSI-4.0b6-20150814-intel-2015a-mt-maxam7.eb

```diff
@@ -0,0 +1,38 @@
+name = 'PSI'
+version = '4.0b6-20150814'
+versionsuffix = '-mt-maxam7'
+
+homepage = 'http://www.psicode.org/'
+description = """PSI4 is an open-source suite of ab initio quantum chemistry programs designed for
+ efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform
+ computations with more than 2500 basis functions running serially or in parallel."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'usempi': False}
+
+source_urls = ['https://github.com/psi4/psi4public/archive/']
+sources = ['7baee9b8ff6e30b39a7ee627750a518c39bd84f3.tar.gz']
+
+patches = [
+    'PSI-%(version)s-fix-gitversion.patch',
+]
+
+configopts = '-DLIBINT_OPT_AM=7 -DENABLE_PLUGINS=ON -DENABLE_DUMMY_PLUGIN=ON '
+
+python = 'Python'
+pyver = '2.7.10'
+
+dependencies = [
+    (python, pyver),
```

Comment:
For commonly used things like Python, it really helps in trying to get users to load modules that are compatible with each other.

But, I agree this is not an ideal solution.

A cleaner way would be to use a module hierarchy and have Python as one of the hierarchy layers. But then you run into the 'matrix' problem, which is yet to be solved...